content/docs/azure/azureEndtoEnd.md

```diff
@@ -0,0 +1,171 @@
+
+## Set up your environment
+### Download the project files
+This tutorial uses the Azure Pipelines example in the Kubeflow examples repo. You can optionally use a pipeline of your own, but several key steps may differ.
+
+Clone the project files and go to the directory containing the [Azure Pipelines (Tacos and Burritos)](https://github.com/kubeflow/examples/tree/master/pipelines) example:
+```
+git clone https://github.com/kubeflow/examples.git
+cd examples/pipelines/azurepipeline
+```
+As an alternative to cloning, you can download the [Kubeflow examples repository zip file](https://github.com/kubeflow/examples/archive/master.zip).
+## Deploy Kubeflow
+If you don't already have one, create an Azure account. If you have not used Azure services before, you can recieve up to [1 year of free services and free credits.](https://azure.microsoft.com/en-ca/free/)
+
+> Note: that some of the services used in this guide may not be included in the free services, but can be covered by free credits. 
+
+First, install the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest), then follow the instructions in the [guide to deploying Kubeflow on Azure](https://www.kubeflow.org/docs/azure/deploy/install-kubeflow/). 
+> Ensure that the agent size you use has the proper memory and storage requirements. For the Azure Pipelines example, **56 GiB** of memory are needed and **premium storage** must be available. Use [this guide](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes) to choose the right agent size for your deployment. (We chose an agent size of Standard_DS13_v2.)
+## Configuring Azure Resources
+### Create an ML Workspace in Azure
+An ML Workspace in Azure is where all of your ML models will be stored and updated. There is also support for managing your active deployments, which will be displayed later in this tutorial.
+
+Go to portal.azure.com and click on your resource group. From there, select the “add a new resource” option. Search for ‘Machine Learning Studio Workspace’ and use the default options, taking note of the name you decide for it. 
+
+![Creating an ML workspace](images/creatingWS.PNG)
+
+### Create an Azure Container Registry
+Go to portal.azure.com and click on your resource group. From there, select the “add a new resource” option. Search for ‘Container Registry’ and add it to your resource group.
+
+Configure your registry by selecting and noting the name you use for it. Enable an admin user, and change the SKU option to ‘Premium’. 
+
+![Create Container Reg](images/createContainerReg.PNG)
+
+### Create a Persistent Volume Claim
+A persistent volumes claim is a dynamically provisioned storage resource attached to a Kubernetes cluster. It is used in the pipeline to store data and files across pipeline steps. 
+
+Using a bash shell, navigate to the azurepipeline directory. Use the following commands to create a persistent volumes claim for your cluster.
```

Comment:
For clarity, put "azurepipeline" in code formatting, like this:

```
`azurepipeline`
```

Do the same for other words/phrases that indicate directory names, code, etc - anything that isn't plain old English.